nvflare/private/fed/utils/fed_utils.py

```diff
@@ -210,7 +210,7 @@ def fobs_initialize(workspace: Workspace = None, job_id: str = None):
 
 def custom_fobs_initialize(workspace: Workspace = None, job_id: str = None):
     if workspace:
-        site_custom_dir = workspace.get_client_custom_dir()
+        site_custom_dir = workspace.get_client_custom_dir(job_id)
```

Comment:
No, this code was meant to register decomposers from the site's local/custom dir.
If you need to register decomposers from the job's custom dir, you need to add code to register workspace.get_app_custom_dir().